### ISSUE TITLE
[Supervisions] Alerts 

### ISSUE THREAD

**Issue by lucychambers** [eHealthAfrica/connect#109]
Create a basic, visual only 'alert' system. Analyse data from ODK Collect, in combination with the acceptable thresholds and flag in red any time a particular metric crosses a particular threshold. 

*Visual only* means that we will not push any notifications outside of the system - as we currently lack information on how the alerts will function. 

For the time being, those using the system will have to log in, in order to see the alerts.

**Comment by grillazz** [comment]
any ODK api exist ? i never touch ODK so all information will be appreciated :)

**Comment by danse** [comment]
@grillazz yes, but the API will be needed when we will need to support automated ingestion, which is tracked by this other ticket https://github.com/eHealthAfrica/connect/issues/114. For this deadline we will just do batch processing and serve static JSON

**Comment by grillazz** [comment]
@danse np with serving static JSON.
I just want to be fimiliar with this api once moment came ;)

**Comment by janetbutts** [comment]
Johannes on NutSurv has already used a webhook to consume formhub data for the NurSurv project. He may be a good resource to use.

**Comment by johanneswilm** [comment]
Hey, yes, feel free to ask me anything about that.

**Comment by danse** [comment]
Cool, thanks

**Comment by grillazz** [comment]
Hey @johanneswilm where we can find in NutSurv webhooks to consume formhubs ? Is this on Python/Django level ?

**Comment by johanneswilm** [comment]
Hey @grillazz yes indeed, python/django level. It needs to be done on the server side, but it could also be done in something else than django/python. I think the most simple example that shows how to do it is this repo: https://github.com/eHealthAfrica/formhub-sms-webhook . In NutSurv you can find the entry point here: https://github.com/eHealthAfrica/eha-nutsurv-django/blob/develop/nutsurv/importer/views.py#L55 . Most of it should be pretty simple and also understandable for non-python developers, but please let me know if there is something you don't understand or want help with.

**Comment by grillazz** [comment]
Thx @johanneswilm Python/Django level is perfect to me :)

**Comment by lucychambers** [comment]
[Mockup v1](https://drive.google.com/open?id=0B40AhgsIbVk2ajNrYXBKVzB5WUk&authuser=0)

Features that would be nice to have: 

* Highlight on the navigation tab which is selected so it is easier to see that you are viewing the RISS reports vs others. 
* Visible filter button to make it clearer that it is possible to filter the data by different columns. Would also be great if the *default* view of the page had the facilities which were underperforming highlighted. 
* Update text in second column: LGA name -> LGA code. 
* Change bars for underperforming facilities to orange (Kano Connect line drawn in: #E1272A - could use this if we want colours to be consistent).

**Comment by danse** [comment]
@grillazz @lucychambers this is done, is not it?

**Comment by lucychambers** [comment]
Hi @danse I highlighted in bold one thing that I think could still be improved, namely that I think the page should default to either the underperforming / best performing filter - depending on how you want to look at it. 

There were some small tweaks to the UI which I also suggested in [Mockup V2](https://drive.google.com/drive/#folders/0B40AhgsIbVk2fklYeEFjLWNSekpXTUpZS2E3WExEMXZDOE5NZk8xYWoyU2RJRDFURlNDalk/0B40AhgsIbVk2fnpxV0VFTzNyVUNtVk5hdHc1eHJHTG5uN2wwV0tlM1NnSE9NWjhNY1lqdjg/0B40AhgsIbVk2fmRkMkRoZzNCMHp5aURoUE5Bb0NsNjY3WTJwSEFFS0lzUmV1OTZHbHNWdjg) e.g. the tab going orange when it was selected. 

But the main essence of the issue is done, we have something to demo!
